docs/source/release_notes.rst

```diff
@@ -11,6 +11,7 @@ Future Release
         * Add autonormalize as an add-on library (:pr:`1840`)
         * Add DateToHoliday Transform Primitive (:pr:`1848`)
     * Documentation Changes
+        * Bump ipython version (:pr:`1857`)
```

Comment:
```suggestion
        * Bump minimum ipython version (:pr:`1857`)
```

Reply:
Strictly speaking we pin the ipython version of the docs

Reply:
okay